src/vs/code/browser/workbench/workbench.ts

```diff
@@ -10,7 +10,9 @@ import { Emitter } from 'vs/base/common/event';
 import { Disposable, IDisposable } from 'vs/base/common/lifecycle';
 import { parse } from 'vs/base/common/marshalling';
 import { Schemas } from 'vs/base/common/network';
-import { posix } from 'vs/base/common/path';
+// --- Start PWB ---
+import { join, posix } from 'vs/base/common/path';
```

Comment:
To reduce merge conflicts you should import `join` on its own line (use an eslint disable to make typescript shut up about two different import statements referencing the same module)

Reply:
ah thank you! I have fixed this now 👍